src/analytics/internal/analytics.js

```diff
@@ -0,0 +1,148 @@
+import { MapEventTypeToInt, MapNotifTypeToIntArray } from './constants'
+import sendToServer from './sendToServer'
```

Comment:
Instead of creating an instance of the `SendToServer` class within its directory and exporting it, I think a better pattern would be to accept it via this `Analytics` class constructor and then, when you create an `Analytics` instance in `src/analytics/internal/index.js` module, pass in either an instance or the class and assign it as a property on the `Analytics` instance. The less calls to outside code, the better (then we can easily pass in different implementations for testing, or whatever other re-use, when we create an instance of `Analytics`). You could also do the `remoteFunction` setup inside the constructor and assign them to instance props.